lib/msal-core/src/utils/RequestUtils.ts

```diff
@@ -34,20 +34,24 @@ export class RequestUtils {
      *
      * validates all request parameters and generates a consumable request object
      */
-    static validateRequest(request: AuthenticationParameters, isLoginCall: boolean, clientId: string, interactionType: InteractionType): AuthenticationParameters {
-
+    static validateRequest(request: AuthenticationParameters, clientId: string, interactionType: InteractionType): AuthenticationParameters {
         // Throw error if request is empty for acquire * calls
-        if(!isLoginCall && !request) {
+        if (!request) {
             throw ClientConfigurationError.createEmptyRequestError();
         }
+        
+        // Check if sent scopes are Login scopes
+
+        // const isLoginScopes = ScopeSet.isLoginScopes(request.scopes, clientId);
```

Comment:
Can you remove these commented out lines?

Reply:
Done